types.proto

```diff
@@ -32,13 +32,25 @@ message HashType {
     UNSPECIFIED = 0;
     SHA256 = 1;
     SHA512 = 2;
+    MD5 = 3;
   }
   HashMethod method = 1;
   bytes hash = 2;
 }
 
-// OpenConfig path representation of the target.
-// eg. /interfaces/interface[name=et-1/0/0]
+// Path encodes a data tree path as a series of repeated strings, with
+// each element of the path representing a data tree node name and the
+// associated attributes.
+// Reference: gNMI Specification Section 2.2.2.
 message Path {
```

Comment:
Is there a reason not to have a dependency between gNOI and gNMI here? Should we move to a common Path definition?